ambari-server/src/main/java/org/apache/ambari/server/events/publishers/AgentCommandsPublisher.java

```diff
@@ -111,7 +111,10 @@ public void sendAgentCommand(Multimap<Long, AgentCommand> agentCommands) throws
                 if (!clusterDesiredConfigs.containsKey(clusterId)) {
                   clusterDesiredConfigs.put(clusterId, clusters.getCluster(clusterId).getDesiredConfigs());
                 }
-              } catch (NumberFormatException|AmbariException ignored) {}
+              } catch (NumberFormatException|AmbariException e) {
+                LOG.error("Exception on sendAgentCommand", e);
+                throw new AmbariRuntimeException(e);
```

Comment:
Oh, wait please.
When installing new cluster, `NumberFormatException` raises because there is no cluster id for new cluster yet on line `clusterId = Long.valueOf(((ExecutionCommand)ac).getClusterId());`.
It hangs and cannot proceed next stages.
I'll fix it wait please.

Reply:
@brahmareddybattula 
Hmm... Is it better not to rethrow? Just logging?

Reply:
yes, just logging should be fine, we just need to avoid the hung right.